nengo_gui/components/pointer.py

```diff
@@ -8,17 +7,15 @@
 import numpy as np
 
 from nengo_gui.components.component import Component
+from nengo_gui.components.spa_plot import SpaPlot
 
-
-class Pointer(Component):
+class Pointer(SpaPlot):
     config_defaults = dict(show_pairs=False, **Component.config_defaults)
     def __init__(self, obj, **kwargs):
-        super(Pointer, self).__init__()
-        self.obj = obj
-        self.data = collections.deque()
+        super(Pointer, self).__init__(obj, **kwargs)
+        # the semantic pointer value as set by the user in the GUI
+        # a value of 'None' means do not override
         self.override_target = None
-        self.target = kwargs.get('args', 'default')
-        self.vocab_out = obj.outputs[self.target][1]
         self.vocab_in = obj.inputs[self.target][1]
 
     def attach(self, page, config, uid):
```

Comment:
Why is attach being overridden here?  This looks like it's just redoing exactly what the base class already does.